portafly/src/i18n/locales/en/audience/accounts/listing.yml

```diff
@@ -58,6 +58,8 @@ accounts_table:
  applications_header_description: Column listing applications linked to accounts, on a 1:1 relationship
  state_header: State
  state_header_description: Column listing the state of the accounts
+ created_header: Created at
+ updated_header: Updated at
```

Comment:
```suggestion
 created_header: Created at
 created_header_description: ...
 updated_header: Updated at
 updated_header_description: ...
```

Reply:
But I think we should export the data as is, with the property names instead of translations.

Reply:
this too, @thomasmaas @cntlsn, any comments?

Reply:
We also need to consider that this kind of CSV files will be used by people who are not technical or are expecting to see the same as in the UI.

Reply:
That's a good point

Reply:
But still we're missing the descriptions.

Reply:
I'll leave it to the docs team